lib/timers.js

```diff
@@ -496,14 +496,13 @@ exports.setInterval = function(callback, repeat, arg1, arg2, arg3) {
   return timeout;
 };
 
-exports.clearInterval = function(timer) {
-  if (timer && timer._repeat) {
-    timer._repeat = null;
-    clearTimeout(timer);
-  }
+exports.clearInterval = function clearInterval(timer) {
+  // clearTimeout and clearInterval can be used to clear timers created from
+  // both setTimeout and setInterval, as specified by HTML Living Standard:
+  // https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-setinterval
+  clearTimeout(timer);
```

Comment:
Can't we just set it to the same function instead of creating this wrapper?

Reply:
FWIW I brought up this option for the sake of users who care about the function name for profilers, stack traces and the like. I personally would just go with `exports.clearInterval = clearTimeout;` but I'm not certain how other collaborators might feel about it.